react/public/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="LINC Dashboard"
+    />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```

Comment:
`<link rel="manifest" href="./manifest.json" />` should work here since you are in the same directory